options/db_options.h

```diff
@@ -110,6 +110,7 @@ struct ImmutableDBOptions {
   Logger* logger;
   std::shared_ptr<CompactionService> compaction_service;
   bool enforce_single_del_contracts;
+  bool disable_delete_obsolete_files_on_open;
```

Comment:
I think you also need to add it here: https://github.com/rockset/rocksdb-cloud/blob/5f6d105a6a5a6437ab4045c5a0c00e4e981e1f4a/options/options_helper.cc#L190